src/folio_data_import/UserImport.py

```diff
@@ -492,8 +464,8 @@ async def process_existing_user(self, user_obj) -> Tuple[dict, dict, dict, dict]
                    the existing user object, the existing request preference object (existing_rp),
                    and the existing PU object (existing_pu).
         """
-        rp_obj = user_obj.pop("requestPreference", {})
-        spu_obj = user_obj.pop("servicePointsUser")
+        rp_obj = user_obj.pop("requestPreference", {}) if "requestPreference" in user_obj else {}
```

Comment:
Shouldn't need this conditional here. `user_obj.pop()` with the default value shouldn't have a problem if there's no `requestPreference` object.